snmp/datadog_checks/snmp/data/default_profiles/dell-sonicwall.yaml

```
@@ -0,0 +1,117 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+#  - dell.yaml  # TODO: check hierarchy first
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "dell"  # TODO: move to _dell.yaml
+      serial_number:
+        symbol:
+          MIB: SONICWALL-COMMON-MIB
+          OID: 1.3.6.1.4.1.8741.2.1.1.2.0
+          name: snwlSysSerialNumber
+      version:
+        symbol:
+          MIB: SONICWALL-COMMON-MIB
+          OID: 1.3.6.1.4.1.8741.2.1.1.4.0
+          name: snwlSysROMVersion
+      model:
+        symbol:
+          MIB: SONICWALL-COMMON-MIB
+          OID: 1.3.6.1.4.1.8741.2.1.1.1.0
+          name: snwlSysModel
+      os_version:
+        symbol:
+          MIB: SONICWALL-COMMON-MIB
+          OID: 1.3.6.1.4.1.8741.2.1.1.3.0
+          name: snwlSysFirmwareVersion
```

```suggestion
      version:
        symbol:
          MIB: SONICWALL-COMMON-MIB
          OID: 1.3.6.1.4.1.8741.2.1.1.3.0
          name: snwlSysFirmwareVersion
      model:
        symbol:
          MIB: SONICWALL-COMMON-MIB
          OID: 1.3.6.1.4.1.8741.2.1.1.1.0
          name: snwlSysModel
```

It seems that firmware version might be better mapped to `version` field.

And ROM version doesn't seems to map well to `os_version`, so maybe we can leave it for now.